Fundamentals/Setup/Upgrading/version-specific.md

```diff
@@ -30,14 +42,14 @@ We updated the `Program` class in the `Program.cs` file to the following out of
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
                     webBuilder.UseStaticWebAssets();
-                    webBuilder.UseStartup&lt;Startup&gt;();
+                    webBuilder.UseStartup<Startup>();
```

Comment:
```suggestion
                    webBuilder.UseStartup&lt;Startup&gt;();
```
Our doesn't display the <Startup> as it considers it a code and skips it. To display it in the document, it needs to be written as &lt;Startup&gt; 

See issue https://github.com/umbraco/UmbracoDocs/issues/3956 where I had resolved this as a solution 🙂

Reply:
Ahhh okay, thanks for letting me know 🙂